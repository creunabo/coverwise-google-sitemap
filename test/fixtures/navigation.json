{
  "level1": [{
    "title": "Rejseforsikringer",
    "url": "/rejseforsikringer",
    "level2": [{
      "title": "Forsikringsoversigt",
      "url": "/forsikringsoversigt"
    }, {
      "title": "Enkeltrejseforsikring",
      "url": "/produkt/enkeltrejseforsikring"
    }]
  }, {
    "title": "F.A.Q.",
    "url": "/faq"
  }, {
    "title": "Om Coverwise",
    "url": "/om-coverwise"
  }, {
    "title": "Anmeld skade",
    "url": "/anmeld-skade"
  }]
}
