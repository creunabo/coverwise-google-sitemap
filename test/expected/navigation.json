{
  "level1": [{
    "title": "Rejseforsikringer",
    "url": "/rejseforsikringer",
    "level2": [{
      "title": "Forsikringsoversigt",
      "url": "/forsikringsoversigt"
    }, {
      "title": "Enkeltrejseforsikring",
      "url": "/produkt/enkeltrejseforsikring"
    }, {
      "title": "Årsrejseforsikring",
      "url": "/produkt/aarsrejseforsikring"
    }, {
      "title": "Bilskadeforsikring",
      "url": "/produkt/bilskadeforsikring"
    },
    {
      "title": "Erhvervsrejseforsikring",
      "url": "/produkt/erhvervsrejseforsikring"
    },
    {
      "title": "Slutleverandørsvigt",
      "url": "/produkt/slutleverandoersvigt"
    },
    {
      "title": "Afbestillingsforsikring",
      "url": "/produkt/Afbestillingsforsikring"
    },
    {
      "title": "Backpacker",
      "url": "/rejseforsikring-backpacker"
    }
   ]
  }, {
    "title": "F.A.Q.",
    "url": "/faq"
  }, {
    "title": "Om Coverwise",
    "url": "/om-coverwise"
  }, {
    "title": "Anmeld skade",
    "url": "/anmeld-skade"
  }]
}
